server/data/quotes.json

```
@@ -56,8 +56,8 @@
     "sentence": "One of the benefits of being bicultural is simply the awareness that how you live is not the only way."
   },
   {
-    "author": "George Carlin",
-    "sentence": "“Meow” means “woof” in cat.”"
+    "author": "Dutch Proverb",
+    "sentence": "Who knows the language is at home everywhere.”"
```

No need of adding another quote symbol at the end of the sentence.

```suggestion
    "sentence": "Who knows the language is at home everywhere."
```